### PR TITLE
Updates for BZ1766200 - clarifying TLS version and router cipher suites

### PR DESCRIPTION
**Pull request by lamek**
https://bugzilla.redhat.com/show_bug.cgi?id=1766200

### CODE REVIEW

File: architecture/networking/routes.adoc [hunk 1]
```diff
@@ -151,8 +151,10 @@ ciphers for the connection to be complete:
 See the link:https://wiki.mozilla.org/Security/Server_Side_TLS[Security/Server
 Side TLS] reference guide for more information.
 
-The router defaults to the `intermediate` profile. You can select a different
-profile using the `--ciphers` option when creating a router, or by changing
+By default the router selects the intermediate profile and sets ciphers based on this profile.
```
Comment (kalexand-rh) [inline architecture/networking/routes.adoc]:
```suggestion
By default, the router selects the intermediate profile and sets ciphers based on this profile.
```

File: architecture/networking/routes.adoc [hunk 2]
```diff
@@ -151,8 +151,10 @@ ciphers for the connection to be complete:
...
+When a profile is selected, only the ciphers are set. The TLS version is not governed by the profile.
+
+You can select a different profile using the `--ciphers` option when creating a router, or by changing
```
Comment (kalexand-rh) [inline architecture/networking/routes.adoc]:
```suggestion
You can select a different profile by using the `--ciphers` option when creating a router or by changing
```

File: architecture/networking/routes.adoc [hunk 3]
```diff
@@ -151,8 +151,10 @@ ciphers for the connection to be complete:
 See the link:https://wiki.mozilla.org/Security/Server_Side_TLS[Security/Server
 Side TLS] reference guide for more information.
 
-The router defaults to the `intermediate` profile. You can select a different
-profile using the `--ciphers` option when creating a router, or by changing
+By default the router selects the intermediate profile and sets ciphers based on this profile.
+When a profile is selected, only the ciphers are set. The TLS version is not governed by the profile.
```
Comment (kalexand-rh) [inline architecture/networking/routes.adoc]: Do we provide information about setting the TLS version elsewhere?

**Review by kalexand-rh — COMMENTED** [review]
Couple of picks and a question, then it LGTM.

File: architecture/networking/routes.adoc (hunk 3 above)
Comment (lamek) [reply architecture/networking/routes.adoc]: The link to the Mozilla page has all the version specific info.

In this context (I believe) you're configuring your router to handle connections from clients supporting a specific version of TLS.

So, for example, if you wanted to only support secure connections from clients using TLS 1.3, you'd set the router to the Modern profile, and anyone using a client implementing an older version of TLS would have their connection request rejected.

Comment (lamek) [reply architecture/networking/routes.adoc]: *this link - https://wiki.mozilla.org/Security/Server_Side_TLS

**Pull request merged**